omnigibson/simulator.py

```diff
@@ -103,12 +103,12 @@ def _launch_app():
     # expects the extensions to be reachable in the parent directory of the kit file. We copy on every launch to
     # ensure that the kit file is always up to date.
     assert "EXP_PATH" in os.environ, "The EXP_PATH variable is not set. Are you in an Isaac Sim installed environment?"
-    kit_file = Path(__file__).parent / "omnigibson.kit"
-    kit_file_target = Path(os.environ["EXP_PATH"]) / "omnigibson.kit"
-    try:
-        shutil.copy(kit_file, kit_file_target)
-    except Exception as e:
-        raise e from ValueError("Failed to copy omnigibson.kit to Isaac Sim apps directory.")
+    # kit_file = Path(__file__).parent / "omnigibson.kit"
+    # kit_file_target = Path(os.environ["EXP_PATH"]) / "omnigibson.kit"
+    # try:
+    #     shutil.copy(kit_file, kit_file_target)
+    # except Exception as e:
+    #     raise e from ValueError("Failed to copy omnigibson.kit to Isaac Sim apps directory.")
```

Comment:
We need to update the kit file. To do so, diff the current kit file with the existing kit file in Isaac Sim 2023.1.1's apps directory (its the python one in there) and get a .patch. Then apply the patch onto the new version's kit file. Include the patch in the repo here so it's easy to apply to future versions.